extensions/VisualEditor/ApiVisualEditor.php

```diff
@@ -133,6 +133,24 @@ protected function requestParsoid( $method, $title, $params ) {
 		return $req->getContent();
 	}
 
+
+	protected function getPreloadHTML( $params ) {
+		$preloadTitle = Title::newFromText( urldecode( $params['preload'] ) );
```

Comment:
I believe this should still work without urldecode - just like it works here - https://github.com/Wikia/app/blob/INT-304/extensions/VisualEditor/ApiVisualEditor.php#L366. Is it possible that it's somehow encoded twice?